python/fusion_engine_client/messages/configuration.py

```diff
@@ -783,18 +785,23 @@ class SetMessageRate(MessagePayload):
     def __init__(self,
                  output_interface: InterfaceID = None,
                  protocol: ProtocolType = ProtocolType.INVALID,
-                 message_id: int = ALL_MESSAGES_ID,
+                 message_id: int = None,
```

Comment:
This is a Python typing violation. `Optional[int]` would be the appropriate type.

Reply:
I'm also a little confused why you changed this. Typically, the pattern you're using here is done when providing a mutable object as a default parameter value (initializing a list to None instead of []) which is to avoid accidental mutations of the object, but I don't think that's a concern here.